frontend/src/api/tus.ts

```diff
@@ -207,6 +205,9 @@ export function abortAllUploads() {
     }
     if (CURRENT_UPLOAD_LIST[filePath].upload) {
       CURRENT_UPLOAD_LIST[filePath].upload.abort(true);
+      CURRENT_UPLOAD_LIST[filePath].upload.options!.onError!(
+        new Error("Upload aborted")
```

Comment:
After aborting an upload, the confirmation prompt is being wrongly displayed when closing the browser tab. This change fixes this issue by rejecting the upload promise, which was hanging previously.